node/consensus/consensus.go

```diff
@@ -437,7 +437,8 @@ func (c *Consensus) SignProposal(proposal smartbft_types.Proposal, _ []byte) *sm
 // (from SmartBFT API)
 func (c *Consensus) AssembleProposal(metadata []byte, requests [][]byte) smartbft_types.Proposal {
 	c.stateLock.Lock()
-	newState, attestations := c.Arma.SimulateStateTransition(c.State, requests)
+	// TODO: config request not handled yet
+	newState, attestations, _ := c.Arma.SimulateStateTransition(c.State, requests)
```

Comment:
Extract the config request and do a log.Info on it if it exists.
Log there it is not supported yet.